### PR TITLE
Modify error message when filtering classes

### PR DESCRIPTION
**Pull request by rakeshgm**
Modify error message when filtering replicationClasses and SnapshotClasses

### CODE REVIEW

File: internal/controller/volumereplicationgroup_controller.go [hunk 1]
```diff
@@ -881,7 +881,7 @@ func (v *VRGInstance) separatePVCUsingPeerClassAndSC(peerClasses []ramendrv1alph
 				return nil
 			}
 
-			return fmt.Errorf("failed to find matching peerClass for PVC") // TODO: better error message
+			return fmt.Errorf("failed to find replicationClass matching peerClass for PVC")
```
Comment (BenamarMk) [inline internal/controller/volumereplicationgroup_controller.go:884]: Maybe it is good to add PVC name as well (if it is not redundant somewhere else)?

File: internal/controller/volumereplicationgroup_controller.go [hunk 2]
```diff
@@ -895,7 +895,7 @@ func (v *VRGInstance) separatePVCUsingPeerClassAndSC(peerClasses []ramendrv1alph
 	}
 
 	if snapClass == nil {
-		return fmt.Errorf("failed to find matching peerClass for PVC") // TODO: better error message
+		return fmt.Errorf("failed to find snapshotClass for PVC")
```
Comment (BenamarMk) [inline internal/controller/volumereplicationgroup_controller.go:898]: Same here.

File: internal/controller/volumereplicationgroup_controller.go (hunk 1 above)
Comment (rakeshgm) [reply internal/controller/volumereplicationgroup_controller.go]: done.

File: internal/controller/volumereplicationgroup_controller.go (hunk 2 above)
Comment (rakeshgm) [reply internal/controller/volumereplicationgroup_controller.go]: done

**Review by ShyamsundarR — APPROVED** [review]

**Pull request merged**